source.txt

```diff
@@ -166,8 +166,10 @@ Table of Contents
 
         URI_ENCODE( <parent_folder> <folder_name> '/' )
 
-    for a folder. Item names MAY contain all characters except '/' and
-    the null character, and MUST NOT have zero length.
+    for a folder. Item names MUST only consist of unreserved characters,
+    as defined in Section 2.3 of [URI]. Item names MUST NOT be equal to
```

Comment:
No, item names are the names as displayed in the folder listing. They are not the strings that end up between /'s in the URL. Reserved characters in item names are totally fine, as long as you encode them! :)